src/app/pages/swap/components/swap-asset-dialog/swap-asset-dialog-base.tsx

```diff
@@ -0,0 +1,33 @@
+import { RouteUrls } from '@shared/route-urls';
```

Comment:
This seems to be almost the same as `SwapAssetDialogQuote`. Maybe we can make them the same a pass a prop to change between `from` and `to` ?

Reply:
This is how it was, I changed it and prefer it as two separate components.